cli/src/main/java/org/crsh/cli/impl/Delimiter.java

```diff
@@ -35,6 +35,8 @@ public void escape(CharSequence s, int start, int end, Appendable appendable) th
           case '\'':
           case '\\':
             appendable.append('\\');
+          default:
```

Comment:
I don't think that these are necessary.  These cases are meant to recognize characters that need to be escaped, and fall through to the point where an escaped backslash is appended before the character at the current point.  Characters that hit the various cases become escaped, while other characters are appended.  a case of "default: break;" has no appreciable effect on the code.  All code should *do* something, and the result of not adding this is the same as adding it.  A default case is appropriate for certain other situations -- maybe you are expecting everything to conform to a set of cases, and you need to throw an error or perform an action when something unexpected is encountered.  But this is not the case for any of the switch statements in this file that perform this type of function.